examples/ga_peoplecount/README.md

```diff
@@ -23,21 +22,29 @@ const sdk = new HuddlySdk(usbApi);
 async function init() {
   await sdk.init();
 
-  sdk.on('ATTACH', (cameraManager) => {
+  sdk.on('ATTACH', async (cameraManager) => {
+    const autozoomCtl = cameraManager.getAutozoomCtl();
+    // Make sure that autozoom (genius framing is started)
+    await autozoomCtl.init();
```

Comment:
In this example you actually want to use the detector right, no need for AZ

Reply:
Yes, but just making sure that autozoom is enabled/started before we subscribe for detections. Might lead to confusion if you setup detection listener but autozoom is not started on the camera. That was my thinking.